### DISCUSSION TITLE
Mask in SVG

### DISCUSSION THREAD

**Discussion by Relison** [WebsiteBeaver/react-native-document-scanner-plugin#66] [Q&A]
Is there a way to put a mask in SVG when viewing the camera?
I'm using zIndex, but even so it doesn't stay in front of the camera

**Comment by Relison** [comment]
![overlay](https://user-images.githubusercontent.com/4913395/228840433-e6b2006d-eb19-4452-880e-63ec85d17bd4.svg)
It would be to put this mask in front of the camera.
Thanks!

**Reply by mvanroon** [reply]
You can use https://github.com/react-native-masked-view/masked-view to add a mask

**Reply by Relison** [reply]
Whenever I use any mask, it stays behind the camera, but the react-native-document-scanner plugin seems to open on top of the application.
